lib/recipes-data/src/lib/parameters.ts

```diff
@@ -1,13 +1,31 @@
-export function mandatoryParameter(name: string): string {
-	if (process.env[name]) {
-		return process.env[name] as string;
-	} else {
-		if (process.env['CI']) {
-			return 'test';
+export function createGetMandatoryParameter<AliasType extends string>(
+	name: string,
+): () => AliasType {
```

Comment:
Question - what's the advantage of defining `AliasType` here, when the type of the return value is guaranteed to be a concrete `string` (either the result of `process.env[name]` or `'test'`)?

Reply:
No advantage, this was me forgetting that in Typescript it's 🦆  all the way down: removed in 4185046f4c5b70dc0f8fdde462a26c754484cd7f